resources/js/oe_whitelabel.project_status.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file
+ * Attaches behaviors for the project status element.
+ */
+(function (bootstrap, Drupal, $) {
+
+  const colorClasses = [
+    'bg-secondary',
+    'bg-info',
+    'bg-dark',
+  ];
+
+  /**
+   * @param {int} msBegin
+   *   Start timestamp in milliseconds.
+   * @param {int} msEnd
+   *   End timestamp in milliseconds.
+   * @param {function(0|1|2)} setStatus
+   *   Callback to set the status: 0 = planned, 1 = ongoing, 2 = closed.
+   */
+  function animateStatus(msBegin, msEnd, setStatus) {
+    const msNow = Date.now();
+    if (msNow < msBegin) {
+      setStatus(0);
+      window.setTimeout(setStatus, msBegin - msNow, 1);
```

Comment:
> We don't have requirements for this, so we should not have done it in the first place
True. My mistake.

> we should leave such work to the FRONT and UX, that is their department and such things need to be requested and validated by business first in the UX side
I would agree even without the animation, because now our stuff depends on selectors coming from front-end.
It was not my decision.